plugin-annotation/src/main/java/com/mapbox/mapboxsdk/plugins/annotation/SymbolManager.java

```diff
@@ -96,6 +103,56 @@ public SymbolManager(@NonNull MapView mapView, @NonNull MapboxMap mapboxMap, @No
   @VisibleForTesting
   SymbolManager(@NonNull MapView mapView, @NonNull MapboxMap mapboxMap, @NonNull Style style, @NonNull CoreElementProvider<SymbolLayer> coreElementProvider, @Nullable String belowLayerId, @Nullable GeoJsonOptions geoJsonOptions, DraggableAnnotationController<Symbol, OnSymbolDragListener> draggableAnnotationController) {
     super(mapView, mapboxMap, style, coreElementProvider, draggableAnnotationController, belowLayerId, geoJsonOptions);
+    mapView.addOnStyleImageMissingListener(new ImageLoader(mapView));
+  }
+
```

Comment:
It's pretty clear from the code, but perhaps add 2-3 javadoc sentences that explain what this class is about